base/uk.ac.stfc.isis.ibex.ui.banner/src/uk/ac/stfc/isis/ibex/ui/banner/models/BannerItemModel.java

```diff
@@ -37,57 +38,92 @@
  */
 public class BannerItemModel extends ModelObject implements IndicatorModel {
 
-    private SettableUpdatedValue<String> text;
-    private SettableUpdatedValue<Color> color;
-    private SettableUpdatedValue<Boolean> availability;
-    private BannerItemViewState converter;
-
+    private final SettableUpdatedValue<String> text = new SettableUpdatedValue<>();
+    private final SettableUpdatedValue<Boolean> availability = new SettableUpdatedValue<>(true);
+    private final SettableUpdatedValue<Color> colour = new SettableUpdatedValue<>(IndicatorColours.BLACK);
+    
+    private final BannerItem item;
+    
+    private static final int MAX_TEXT_LENGTH = 30;
+    private static final String ELIPSES = "...";
```

Comment:
Please could you change this variable name.

- Ellipsis has two 'l's
- This variable is for one ellipsis (i.e. sequence of dots), not multiple ellipses